fastify.js

```diff
@@ -348,6 +348,13 @@ function fastify (options) {
   }
 
   Object.defineProperties(fastify, {
+    listeningOrigin: {
+      get () {
+        const addresses = this.addresses()
+        const address = addresses[addresses.length - 1]
```

Comment:
```suggestion
        const address = this.addresses().slice(-1).pop()
```

Reply:
I like your 1 line solution, but seems to me it adds a bit of cognitive effort, I mean the 2 lines solution is more explicit. What do you think? 
Maybe it's just my impression...

Reply:
I think this is down to familiarity with the functional `Array` API. I think the two versions have an equal tradeoff in terms of cognitive load.

Another version could be:

```js
const address = this.addresses().at(-1)
```

However, that requires at least Node.js 16.6.0. So we can't use that until the next major version of Fastify.

Reply:
Ok, I'll take your advice, thanks